RPS src/app.js

```diff
@@ -0,0 +1,53 @@
+import compPlay from './RPS.js';
+import rpsCompair from './RPScomp.js';
+
+const fight = document.getElementById('fight');
+
+const playerImg = document.getElementById('playerMove');
+const pow = document.getElementById('pow');
+const compImg = document.getElementById('compMove');
+const win = document.getElementById('win');
+const lose = document.getElementById('loss');
+
+
+let humanScore = 0;
+let computerScore = 0;
+
+fight.addEventListener('click', () => {
+    
+    compImg.classList.remove('hidden');
+    pow.classList.remove('hidden');
+    
+    const compMove = compPlay();
```

Comment:
you should maybe remove the hidden class after you set the source for the file (though the code reads and changes so fast I doubt anyone would ever notice)